shinken/objects/service.py

```diff
@@ -543,6 +543,19 @@ class Service(SchedulingItem):
 #                        |___/
 ######
 
+    def __repr__(self):
+        return '<Service host_name=%r desc=%r name=%r use=%r />' % (
+            getattr(self, 'host_name', None),
+            getattr(self, 'service_description', None),
+            getattr(self, 'name', None),
+            getattr(self, 'use', None)
+        )
```

Comment:
greatly help for debug purpose, when you're in a debugger ;)
and this doesn't change anything hopefully (as we don't expect str_or_repr(a_service_or_a_host) to return something well defined).

So same answer : I won't make a separate PR for this :p